modular_boh/code/game/ranks/vesta_ranks.dm

```diff
@@ -580,32 +580,32 @@
 
 
 /datum/mil_rank/marine_corps/w1
-	name = "Warrant Officer (W-1)"
-	name_short = "WO"
+	name = "Second Warrant Officer (W-1)"
+	name_short = "2ndWO"
 	accessory = list(/obj/item/clothing/accessory/solgov/rank/marine_corps/warrant_officer/w1)
 	sort_order = 11
 
 /datum/mil_rank/marine_corps/w2
-	name = "Second Warrant Officer (W-2)"
-	name_short = "SWO"
+	name = "First Warrant Officer (W-2)"
+	name_short = "1stWO"
 	accessory = list(/obj/item/clothing/accessory/solgov/rank/marine_corps/warrant_officer/w2)
 	sort_order = 12
 
 /datum/mil_rank/marine_corps/w3
-	name = "First Warrant Officer (W-3)"
-	name_short = "FWO"
+	name = "Master Warrant Officer (W-3)"
+	name_short = "MWO"
 	accessory = list(/obj/item/clothing/accessory/solgov/rank/marine_corps/warrant_officer/w3)
 	sort_order = 13
 
 /datum/mil_rank/marine_corps/w4
 	name = "Major Warrant Officer (W-4)"
-	name_short = "MWO"
+	name_short = "MajWO"
 	accessory = list(/obj/item/clothing/accessory/solgov/rank/marine_corps/warrant_officer/w4)
 	sort_order = 14
 
 /datum/mil_rank/marine_corps/w5
-	name = "General Warrant Officer (W-5)"
-	name_short = "GWO"
+	name = "Major Warrant Officer of the Marine Corps (W-5)"
```

Comment:
small misconception with this one here. "General Warrant Officer" would be W-5. The *alternate* of W-5 would be "General Warrant Officer of the Marine Corps".

This is similar to how there's "Sergeant Major" and "Sergeant Major of the Marine Corps"

Reply:
The original idea was just to mimmick SEA ranks. Fleet has W-3 Senior Chief Warrant Officer, W-4 Master Chief Warrant Officer, and W-5 Command Master Chief Warrant Officer. Which is just E-8, E-9, and E-9 alt respectively. 

With MWOMC, Marine WO ranks would follow that same structure of E-8, E-9, E-9 alt. However that being said, I literally don't care what we name W-4+ because the likelihood it's ever seen in a real game is slim to none. We could name it "Poop Shit Cum Warrant Officer" and it would most likely change nothing because when would you ever see a W-5 in game??

Will commit a change in just a minute.